### PR TITLE
kola: write proper fleet test that waits for etcd availability

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: kola/tests/coretest/fleet.go [hunk 1]
```diff
@@ -73,37 +35,39 @@ func TestFleetctlRunService() error {
...
 
-	stdout, stderr, err = timeoutFleetctl("destroy", serviceName)
+	stdout, stderr, err = Run("fleetctl", "destroy", serviceName)
 	if err != nil {
 		return fmt.Errorf("fleetctl destroy failed with error: %v\nstdout: %s\nstderr: %s", err, stdout, stderr)
 	}
-	return nil
-}
 
-func timeoutFleetctl(action string, unitName string) (stdout string, stderr string, err error) {
```
Comment (pbx0) [inline kola/tests/coretest/fleet.go]: Is it possible that without this we would hang indefinitely on the fleetctl?

Comment (mischief) [inline kola/tests/coretest/fleet.go]: it already leaks processes on the system if it timed out anyway, since we never send kill if the timer expires...

**Comment by pbx0** [comment]
LGTM if it seems to work in your testing. It seems like the right change. Any subsequent fleet failures that may happen we can look into when they happen.

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by pbx0** [comment]
lgtm

**Pull request merged**